modules/data.land/R/match_species_id.R

```diff
@@ -0,0 +1,65 @@
+#' Match BETY species ID.
+#'
+#' Parses species codes in input data and matches them with the BETY species ID.
+#' 
+#' @param input_codes Character vector of species codes
+#' @param format_name Species code format name (character, length 1)
+#' @param betydb \code{dplyr} \code{src} object containing BETY connection 
```

Comment:
I'm not saying to build the dplyr from con, or to stop using dplyr, but just to make sure that we're 100% consistent in what we call that connection variable.